magefiles/magefile.go

```diff
@@ -267,15 +267,14 @@ func (ci CI) setRequiredEnvVars() error {
 				envVarPrefix = "HAS"
 				imageTagSuffix = "has-image"
 				testSuiteLabel = "has,e2e-demo"
-			case strings.Contains(jobName, "jvm-build-service"):
-				envVarPrefix = "JVM_BUILD_SERVICE"
-				imageTagSuffix = "jvm-build-service-image"
-				testSuiteLabel = "jvm-build"
 			case strings.Contains(jobName, "release-service"):
 				envVarPrefix = "RELEASE_SERVICE"
 				imageTagSuffix = "release-service-image"
 				testSuiteLabel = "release"
-
+			case strings.Contains(jobName, "jvm-build-service"):
+				envVarPrefix = "JVM_BUILD_SERVICE"
+				imageTagSuffix = "jvm-build-service-image"
+				testSuiteLabel = "jvm-build"
 				// Since CI requires to have default values for dependency images
```

Comment:
the code that's part of this case belongs only to jvm-build-service

Reply:
Ah, sorry about this, I totally missed the fact that it's all indented and part of the jvm-build-service case.

Reply:
no problem, I approved that PR! 😬